src/migration/1689590210380-NotificationPreference.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
Currently looking at why the migration didn't include the FKs to cascade on delete.